main/menu.js

```diff
@@ -144,7 +156,12 @@ function showOpenTorrentAddress () {
 function getAppMenuTemplate () {
   var fileMenu = [
     {
-      label: 'Create New Torrent...',
+      label: 'Create New Torrent from File',
+      accelerator: 'CmdOrCtrl+F',
+      click: showOpenSeedFile
+    },
+    {
+      label: 'Create New Torrent from Folder',
```

Comment:
Need a conditional here. Should still be 'Create New Torrent...' on OS X.

Add '...' at the end for non OS X.